support-frontend/cloud-formation/cfn.yaml

```diff
@@ -319,6 +317,24 @@ Resources:
     - TargetGroup
     - ElasticLoadBalancer
 
+  CatalogLoadingFailureAlarm:
+    Type: AWS::CloudWatch::Alarm
+    Condition: CreateProdResources
+    Properties:
+      AlarmActions:
```

Comment:
Might be worth adding an [AlarmDescription](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-cw-alarm.html#cfn-cloudwatch-alarms-alarmdescription) for new engineers who may not understand the impact of this?